### PR TITLE
User Profile

### PR DESCRIPTION
**Pull request by HoangThiTuanDung**

### CODE REVIEW

File: app/Controller/UsersController.php [hunk 1]
```diff
@@ -125,4 +133,54 @@ public function team($id) {
...
+		$user = $this->User->findById($this->Auth->user('id'));
+		if ($this->request->is('post')) {
+			$data = $this->request->data;
+			if(!empty($data['User']['image']['name'])) {
+				$file = $data['User']['image'];
+				$ext = substr(strtolower(strrchr($file['name'], '.')), 1);
+				$arrExt = ['jpg', 'jpeg', 'gif', 'png'];
+				if(in_array($ext, $arrExt)) {
+					move_uploaded_file($file['tmp_name'], WWW_ROOT . 'img/' . $file['name']);
+
```
Comment (tranluan91) [inline app/Controller/UsersController.php]: remove unnecessary line

**Comment by tranluan91** [comment]
ok

File: app/Controller/UsersController.php [hunk 2]
```diff
@@ -125,4 +133,53 @@ public function team($id) {
...
 		$this->response->body(json_encode($teamOptions));
 	}
+
+	public function profile() {
+		$user = $this->User->findById($this->Auth->user('id'));
+		if ($this->request->is('post')) {
+			$data = $this->request->data;
+			if(!empty($data['User']['image']['name'])) {
+				$file = $data['User']['image'];
+				$ext = substr(strtolower(strrchr($file['name'], '.')), 1);
```
Comment (dinhhoanglong91) [inline app/Controller/UsersController.php]: Please use ```pathinfo``` function 
```php
$ext = pathinfo($file['name'], PATHINFO_EXTENSION);
```

File: app/Controller/UsersController.php [hunk 3]
```diff
@@ -125,4 +133,53 @@ public function team($id) {
...
+				$arrExt = ['jpg', 'jpeg', 'gif', 'png'];
```
Comment (dinhhoanglong91) [inline app/Controller/UsersController.php]: Variable name isn't good. ```$valid_extensions``` is better.

File: app/Controller/UsersController.php [hunk 4]
```diff
@@ -125,4 +133,53 @@ public function team($id) {
...
+				if(in_array($ext, $arrExt)) {
```
Comment (dinhhoanglong91) [inline app/Controller/UsersController.php]: Lack of space after if.

File: app/View/Users/change_password.ctp [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<div class="container">
+	<div class="alert alert-danger flash">
+		<?php echo $this->Session->flash(); ?>
+	</div>
+	<h2>Change Password</h2>
```
Comment (dinhhoanglong91) [inline app/View/Users/change_password.ctp]: Lack of ```__```

**Comment by dinhhoanglong91** [comment]
OK

**Pull request merged**